.github/CODEOWNERS

```diff
@@ -332,6 +332,7 @@
 /packages/kbn-handlebars/ @elastic/kibana-security
 /src/core/server/http/csp/ @elastic/kibana-security @elastic/kibana-core
 /src/plugins/interactive_setup/ @elastic/kibana-security
+/src/plugins/telemetry/server/config/telemetry_labels.ts @elastic/kibana-security
```

Comment:
I moved the validation of the labels to a separate file so I can ping the security team to validate any security concerns 😇